x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/index.tsx

```diff
@@ -38,32 +37,76 @@ const NoTransactionsTitle = styled.span`
   font-weight: bold;
 `;
 
-const TransactionBreakdown: React.FC = () => {
-  const [showChart, setShowChart] = useState(false);
+const TransactionBreakdown: React.FC<{
+  initialIsOpen?: boolean;
+}> = ({ initialIsOpen }) => {
+  const [showChart, setShowChart] = useState(!!initialIsOpen);
 
   const {
     data,
     status,
     receivedDataDuringLifetime
   } = useTransactionBreakdown();
 
-  const kpis = useMemo(
+  const kpis = data ? data.kpis : undefined;
+  const timeseriesPerSubtype = data ? data.timeseries_per_subtype : undefined;
```

Comment:
I see this access checks a lot of places which makes the code a bit harder to read. What about setting a default response like we do in:

https://github.com/elastic/kibana/blob/868c7125c64a4b7b25484a307cde4e9aafa7109f/x-pack/legacy/plugins/apm/public/hooks/useServiceMetricCharts.ts#L13-L15

Could be something like:
```ts
const INITIAL_DATA: TransactionBreakdownAPIResponse = {
      kpis: [],
      timeseries_per_subtype: null
    }
```